src/taipy/config/checker/_checkers/_gLobal_config_checker.py

```diff
@@ -24,11 +24,19 @@ def __init__(self, config: _Config, collector: IssueCollector):
     def _check(self) -> IssueCollector:
         global_config = self._config._global_config
         self._check_clean_entities_enabled_type(global_config)
+        self._check_repository_type(global_config)
         return self._collector
 
+    def _check_repository_type(self, global_config: GlobalAppConfig):
+        value = global_config.repository_type
+        if value != "filesystem" and value != "sql":
+            self._warning(
+                global_config._REPOSITORY_TYPE_KEY,
+                value,
+                f"unknown value: {value} for field {global_config._REPOSITORY_TYPE_KEY}. Default value 'filesystem' is applied.")
```

Comment:
```suggestion
                f'Unknown value "{value}" for field {global_config._REPOSITORY_TYPE_KEY} of GlobalAppConfig. '
                f'Default value "filesystem" is applied.'
            )
```
Just a little bit of reformatting.